test/unit/spec/common/jobs-Spec.js

```diff
@@ -207,9 +208,9 @@ define([
         });
     });
 
-    describe('createJobStatusLines', () => {
+    describe('createJobStatusLines and createJobStatusSummary', () => {
         let container;
-        beforeAll(() => {
+        beforeEach(() => {
             container = document.createElement('div');
```

Comment:
Maybe this should get wiped out in an `afterEach`

Reply:
Though it does look like `.innerHTML = ` gets used each time, so it might be okay. But it's probably better to be safe.

Reply:
will fix this -- thanks!